client/fire-webclient/build.gradle.kts

```diff
@@ -27,8 +27,10 @@ dependencies {
     api(project(CommonsUtil))
     api(project(CommonsHttp))
 
-    implementation(AwsS3K)
-    implementation(AwsS3)
+    implementation(AwsS3K) {
+        exclude("com.squareup.okhttp3:okhttp")
```

Comment:
Is it possible to use a constant for the exclude section as well? Or is it better to leave it explicit for these cases?

Reply:
we can have constant name but then as your name include version we will need to remove it so having two contains one with name and one without